pkgs/applications/video/kodi-cli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, makeWrapper, pkgs, curl, bash, jq, youtube-dl, gnome3 }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.1";
```

Comment:
The changes @symphorien was speaking of for the derivation names isn't in master yet so this should be

`name = "kodi-cli-${version};`
`version = "1.1.1";`

We'll probably just do a big refactor of this anyway so there isn't a need to constrain this.